components/script/script_runtime.rs

```diff
@@ -436,12 +445,29 @@ unsafe fn new_rt_and_cx_with_parent(parent: Option<ParentRuntime>) -> Runtime {
     DisableIncrementalGC(cx);
 
     unsafe extern "C" fn dispatch_to_event_loop(
-        _closure: *mut c_void,
-        _dispatchable: *mut Dispatchable,
+        closure: *mut c_void,
+        dispatchable: *mut JSRunnable,
     ) -> bool {
-        false
+        let networking_task_src: &NetworkingTaskSource = &*(closure as *mut NetworkingTaskSource);
+        let runnable = Runnable(dispatchable);
+        let task = task!(dispatch_to_event_loop_message: move || {
+            runnable.run(RustRuntime::get(), Dispatchable_MaybeShuttingDown::NotShuttingDown);
+        });
+
+        match networking_task_src.queue_unconditionally(task) {
```

Comment:
This can be written as `network_task_src.queue_unconditionally(task).is_ok()`

Reply:
done.